Week-2/2.1/src/index.css

```diff
@@ -0,0 +1,63 @@
+header {
+    background-color: lightgrey;
+    color: #fff;
+    padding: 20px;
+}
+nav {
```

Comment:
-Not a good practice to write css on HTML tags directly. better to use class or id